ci/dev/fmt.sh

```diff
@@ -32,13 +32,14 @@ main() {
   doctoc --title '# iPad' docs/ipad.md > /dev/null
   doctoc --title '# Termux' docs/termux.md > /dev/null
 
-  if [[ ${CI-} && $(git ls-files --other --modified --exclude-standard) ]]; then
-    echo "Files need generation or are formatted incorrectly:"
-    git -c color.ui=always status | grep --color=no '\[31m'
-    echo "Please run the following locally:"
-    echo "  yarn fmt"
-    exit 1
-  fi
+  ## TODO: replace with a method that generates fewer false positives.
```

Comment:
Was it a yarn.lock triggering this? Maybe we are missing a `--frozen-lockfile` somewhere. :thinking: 

I think we should leave this check in because if a yarn.lock is changing that is a bug we want to catch (I assume it means CI may be using different versions than local).

The real bug here is probably the error message. This state can be caused by a lack of formatting, volatile lock files, and possibly other things (although nothing else comes to mind at the moment).